tools/indexbuilder/indexbuilder.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2018 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+// This is a testing tool to inject fake actions to the blockchain
+// To use, run "make build" and " ./bin/indexbuilder"
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	_ "go.uber.org/automaxprocs"
+	"github.com/iotexproject/iotex-core/explorer"
+	"github.com/iotexproject/iotex-core/logger"
+)
+
+func main() {
+	// start block id of the index build
+	var fromBlockID int64
+	// end block id of the index build
+	var toBlockID int64
+	// end point of rds
+	var batchSize int64
+	// retry number
+	var retryNumber int
+	// target address for jrpc connection. Default is "127.0.0.1:14004"
+	var explorerAddr string
+
+	flag.Int64Var(&fromBlockID, "from-block-id", 0, "sync from which block id")
+	flag.Int64Var(&toBlockID, "to-block-id", 0, "sync to which block id")
+	flag.Int64Var(&batchSize, "batch-size", 1, "batch size")
+	flag.IntVar(&retryNumber, "retry-number", 3, "retry number")
+	flag.StringVar(&explorerAddr, "explorer-addr", "127.0.0.1:14004", "target ip:port for jrpc connection")
+	flag.Parse()
+
+	proxy := explorer.NewExplorerProxy("http://" + explorerAddr)
```

Comment:
http or https? how to specify the scheme?

Reply:
you mean how to use it?

Reply:
I don't get the question

Reply:
could we ask user who want to use this command to specify the scheme as part of "explorerAddr"?